### PR TITLE
luakit: add HTTPS support

### PR DESCRIPTION
**Pull request by emmanuelrosa**
###### Motivation for this change

The current `luakit` package doesn't include `glib-networking` as a dependency. Thus when accessing HTTPS sites `luakit` blocks the site, claiming it's _untrusted_.

This change adds `glib-networking` so that `luakit` supports HTTPS.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - ~[ ] macOS~
   - [x] other Linux distributions
- ~[ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))~
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Determined the impact on package closure size (by running `nix path-info -S` before and after): previous package 968318936, new package 1068268984
- [X] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/browsers/luakit/default.nix [hunk 1]
```diff
@@ -27,7 +27,8 @@ in stdenv.mkDerivation rec {
     gst_all_1.gstreamer gst_all_1.gst-plugins-base
     gst_all_1.gst-plugins-good gst_all_1.gst-plugins-bad gst_all_1.gst-plugins-ugly
     gst_all_1.gst-libav
-                ];
+    glib-networking
```
Comment (worldofpeace) [inline pkgs/applications/networking/browsers/luakit/default.nix]: Can you add
```suggestion
    glib-networking # TLS support
```

So we can be clear on why it's there?

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build luakit

**Review by worldofpeace — APPROVED** [review]
I didn't test but it sounds good, I've ran into this issue a few times as well.

**Comment by spiderbit** [comment]
Stupid question is that in the unstable branch?

I added that but did not help so far:

```
    (let newpkgs = import (fetchTarball https://github.com/NixOS/nixpkgs-channels/archive/nixos-unstable.tar.gz) {};
    in
      newpkgs.luakit
    )
```

**Comment by vcunat** [comment]
Yes, it is.  I re-tested this simple pair of commands
```
$ nix build -f https://github.com/NixOS/nixpkgs-channels/archive/nixos-unstable.tar.gz luakit
$ ./result/bin/luakit https://nixos.org
```

**Comment by spiderbit** [comment]
Sorry I am stupid I had a instance of luakit open in the background so it only opened a new window not start the new binary :D works now.

**Pull request merged**